includes/Core/Util/Activation.php

```diff
@@ -155,9 +155,74 @@ private function get_activation_notice() {
 					ob_start();
 					?>
 					<script type="text/javascript">
-					if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
-						sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
-					}
+						if ( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
+							sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
+						}
+
+						jQuery( document ).ready( function( $ ) {
```

Comment:
Let's enqueue `jquery` on the server-side if we use it here. I know it's _probably_ going to be there, but we should do that properly. We can use the existing `admin_enqueue_scripts` hook in the `Activation` class to check whether the activation notice is active on this screen (same logic as the notice uses) and then just enqueue `jquery`.